physionet-django/console/templates/console/event_agreement_new_version.html

```diff
@@ -0,0 +1,18 @@
+{% extends "console/base_console.html" %} {% load static %} {% block title %}{{
+SITE_NAME }} - Event Agreements{% endblock %} {% block content %}
+<div class="card mb-3">
+  <div class="card-header">{{ event_agreement.name }} - new version</div>
+  <div class="card-body">
+    <form
+      method="post"
+      action="{% url 'event_agreement_new_version' event_agreement.pk %}"
+    >
```

Comment:
Not sure if we have a style guide but the Google HTML/CSS one recommends putting the last `>` on the end of the last line with attributes, e.g.

```
<form ...
more text here ...>
```

https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

Reply:
I agree. I always have the tag closed with the attribute. its my bad.